index.html

```diff
@@ -146,7 +194,11 @@ <h4>BREAKING BAD</h4>
           </div>
         </a>
         <!-- Trivia Card -->
-        <a class="card-vertical" id="trivia-card" href="#">
+        <a
+          class="card-vertical"
+          id="trivia-card"
```

Comment:
id attribute for each element should have unique value throughout the page

Reference - https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id